app/templates/services/_base_service_page.html

```diff
@@ -17,45 +17,47 @@
     <div class="column-one-whole">
       {% import "toolkit/summary-table.html" as summary %}
       {% for section in sections %}
-        {{ summary.heading(section.name) }}
-        {% if section.editable %}
-          {% block edit_link scoped %}{% endblock %}
-        {% endif %}
-        {% call(question) summary.list_table(
-          section.questions,
-          caption=section.name,
-          field_headings=[
-            "Service attribute name",
-            "Service attribute",
-            "Add or Edit"
-          ] if section.edit_questions else [
-            "Service attribute name",
-            "Service attribute",
-          ],
-          field_headings_visible=False
-        ) %}
-          {% call summary.row(complete=not question.answer_required) %}
-            {{ summary.field_name(question.label) }}
-            {% if question.answer_required %}
-              {% call summary.field() %}
-                {% if framework and framework.status == 'open' %}
-                    <a href="{{ url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id) }}">Answer required</a>
-                {% else %}
-                  Not answered
-                {% endif %}
-              {% endcall %}
-            {% elif question.is_empty %}
-              {% call summary.field() %}
-                <span class="summary-item-field-answer-required">{{ question.empty_message }}</span>
-              {% endcall %}
-            {% else %}
-              {{ summary[question.type](question.value, question.assurance) }}
-            {% endif %}
-            {% if section.edit_questions %}
-              {{ summary.edit_link('Add' if question.is_empty else 'Edit', url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id, question_slug=question.slug)) }}
-            {% endif %}
+        <div id="{{ section.slug }}">
```

Comment:
It would be cleaner to have `section.heading` take `id` as an optional parameter. Maybe I should not be lazy.